warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/ShardedTableMapFile.java

```diff
@@ -349,7 +350,19 @@ public static Path createShardedMapFile(Logger log, Configuration conf, Path wor
                     Range range = new Range();
                     Locations locations = tableOps.locate(shardedTableName, Collections.singletonList(range));
                     List<TabletId> tabletIds = locations.groupByRange().get(range);
-                    tabletIds.forEach(tId -> splitToLocation.put(tId.getEndRow(), locations.getTabletLocation(tId)));
+                    try {
```

Comment:
Please remove this try block since it will just print superfluous warnings. The underlying class returns an unmodifiable map of Range to unmodifiable list of TableId. removeIf mutates the list, and since you are trying to modify an unmodifiable list, you get an UnsupportedOperationException.